analysis/topEFT/topeft.py

```diff
@@ -160,20 +161,35 @@ def process(self, events):
         cleanedJets['isGood']  = isTightJet(getattr(cleanedJets, jetptname), cleanedJets.eta, cleanedJets.jetId, jetPtCut=30.) # temporary at 25 for synch, TODO: Do we want 30 or 25?
         goodJets = cleanedJets[cleanedJets.isGood]
 
-        # count jets, jet 
+        # Count jets, jet 
         njets = ak.num(goodJets)
         ht = ak.sum(goodJets.pt,axis=-1)
         j0 = goodJets[ak.argmax(goodJets.pt,axis=-1,keepdims=True)]
 
-        # to do: check these numbers are ok
-        if year == "2017": btagwpl = 0.0532 #WP loose 
-        else: btagwpl = 0.0490 #WP loose 
+        # Loose DeepJet WP
+        # TODO: Update these numbers when UL16 is available, and double check UL17 and UL18 at that time as well
+        if year == "2017":
+            btagwpl = get_cut("btag_wp_loose_UL17")
+        elif year == "2018":
+            btagwpl = get_cut("btag_wp_loose_UL18")
+        elif ((year=="2016") or (year=="2016APV")):
```

Comment:
This is fine, but `elif "2016" in year:` also works.

Reply:
Thanks, yep, looks like that would work too. But I suppose that would not catch a typo where the year string includes `2016` (e.g. `20167` would just silently get mapped to 2016, even if you intended it to be `2017` and just accidentally typed a `6` in there for some reason), so I guess for that reason I'd probably prefer to keep checking for exact matches.

Reply:
That's a really good point!